src/main/java/uk/gov/hmcts/reform/hmc/helper/HearingActualsMapper.java

```diff
@@ -73,16 +73,6 @@ private ActualHearingDayEntity toActualHearingDayEntity(ActualHearingDay actualH
 
     private void setNotRequired(ActualHearingDayEntity actualHearingDayEntity, ActualHearingDay actualHearingDay) {
         actualHearingDayEntity.setNotRequired(actualHearingDay.getNotRequired());
-        if (null == actualHearingDay.getNotRequired() || !actualHearingDay.getNotRequired()) {
```

Comment:
Since this is no longer required, this `setNotRequired()` method looks like it now only contains one line - we might as well move this back into the invoking method since there's no additional logic?

Reply:
removed the method